doc/manuals/index.md

```diff
@@ -0,0 +1,33 @@
+# Quantumleap
+
+Quantumleap is an adapter aimed to bring [FIWARE NGSIv2](http://docs.orioncontextbroker.apiary.io/#) Historical Data on top of TimeSeries Databases.
```

Comment:
QuantumLeap is an API that support the storage of NGSI [FIWARE NGSIv2](http://docs.orioncontextbroker.apiary.io/#) data into a [time series database](https://en.wikipedia.org/wiki/Time_series_database).